Modules/socketmodule.c

```diff
@@ -1782,10 +1782,10 @@ getsockaddrarg(PySocketSockObject *s, PyObject *args,
                 Py_TYPE(args)->tp_name);
             return 0;
         }
-        if (!PyArg_ParseTuple(args, "si|iiy*", &interfaceName,
-                              &protoNumber, &pkttype, &hatype,
-                              &haddr))
+        if (!PyArg_ParseTuple(args, "si|iiy*:getsockaddrarg", &interfaceName,
```

Comment:
this change, and every other change where I added ':' and a function name to the format argument of a PyArg_* function, is meant to utilize my patch of seterror() in Python/getargs.c.